docs/clients/promtail/pipelines.md

```diff
@@ -0,0 +1,198 @@
+# Pipelines
+
+A detailed look at how to setup Promtail to process your log lines, including
+extracting metrics and labels.
+
+## Pipeline
+
+A pipeline is used to transform a single log line, its labels, and its
```

Comment:
What happens if multiple pipelines process the same line. Is the line sent to Loki multiple times in that case?